core/src/main/java/bisq/core/offer/CreateOfferService.java

```diff
@@ -294,25 +274,6 @@ public double getSellerSecurityDepositAsDouble(double buyerSecurityDeposit) {
                 Restrictions.getSellerSecurityDepositAsPercent();
     }
 
-    public Coin getMakerFee(Coin amount) {
-        return makerFeeProvider.getMakerFee(bsqWalletService, preferences, amount);
-    }
-
-    public long getMaxTradeLimit(PaymentAccount paymentAccount,
-                                 String currencyCode,
-                                 OfferPayload.Direction direction) {
-        if (paymentAccount != null) {
-            return accountAgeWitnessService.getMyTradeLimit(paymentAccount, currencyCode, direction);
-        } else {
-            return 0;
-        }
-    }
-
-    public boolean isBuyOffer(OfferPayload.Direction direction) {
-        return OfferUtil.isBuyOffer(direction);
-    }
-
-
```

Comment:
3 methods above were moved to OfferUtil.
(test -- can I add inline comment to my own pr?)